templates/base.html

```diff
@@ -13,8 +13,8 @@
         {% endcomment %}
         <link rel="shortcut icon" href='{% static "img/brand/favicon.ico" %}' type="image/x-icon">
         <link rel="icon" href='{% static "img/brand/favicon.ico" %}' type="image/x-icon">
-        {% comment %}replace with image and HTML generated by http://realfavicongenerator.net/ one day{% endcomment %}
```

Comment:
@kerryrm What was this originally supposed to do?

Reply:
That wasn't my comment. It had been there for years.